process/process.md

```diff
@@ -0,0 +1,291 @@
+
+## Verify that your commit includes only relevant changes
+
+Use `git log -p` or GUI tool to check that you are not committing
+extra changes and that you didn't forget to include something.
+
+## Style-check and spell-check your changes
+
+Some projects provide built-in checkers, like `checkpatch` tools,
+which is used in Linux and Zephyr. Use this tool before creating PR.
+
+Also, it is very good idea to configure spell checker in your editor,
+so reviewers will be not busy pointing spelling errors in your
+documentation, comments and commit message.
+
+## Create Pull Request
+
+When your commits are ready, you can create a pull requests to
+appropriate repositories. We are hosting all our code on GitHub, so
+you should be familiar with the process. But, in short:
+
+1. Fork the repository into your account
+2. Clone it locally
+3. Make changes and create commits as described above
+4. Push changes back into your GitHub repository, preferably in a
+   separate branch
+5. Use this branch to create Pull Request into corresponding
+   Xen-Troops repository
+
+### Describe your Pull Request
+
+If your PR consists of one commit, GitHub will do this for you. But if
+there are more commits, you need to provide comprehensive description
+for the PR as a whole. The same rules apply: short and descriptive PR
+name, detailed description of a problem solved. Do not try to squash
+your commit messages, write description for the PR as one entity instead.
+
+If appropriate, write how you tested it. Wait for automated checks to run.
+
+### Check-list
+
+* Code conforms coding style
+* Documentation is up-to date
+* Commits are atomic
+* Commits have good description
+* Code checker / style checker is happy
+* PR description is in good shape
+
+## Review
+
+When you satisfied with your PR, as someone to review it. You can find
+list of maintainers and reviewers at
+[reviewers](https://xen-troops.github.io/reviewers) page. In most
+cases GitHub will assign reviewers immediately, but it is a good idea
+to ping reviewers to tell them that PR is really ready for review.
+
+Not every comment from reviewer require changes in code. But many of
+them are. If reviewer asks you a question, do not haste to change your
+code. Answer the question first.
+
+Every project has maintainer(s). Maintainers have full vision of
+project and its architecture. In most cases maintainers will support
+code you contributed for rest of its life. So maintainers can ask you
+to make changes into your PR so it better fit into project's
+architecture. Even if your code is 100% correct and bug-free, it does
+not mean that it will be accepted as is.
+
+When all comments are addressed, reviewers need to explicitly approve
+pull request by providing Reviewed-by or Acked-by tag.
+
+After collecting all needed R-b tags, you should include them
+into appropriate commit messages, rebase changes onto current upstream
+branch and force push changes into your local branch.
+
```

Comment:
I miss here rules to update pull request when addressing reviewer's comments. Somehow our Renesas colleagues had issue with this in the past: https://github.com/aoscloud/meta-aos-rcar-gen5/pull/2#issuecomment-1959032876